### ISSUE TITLE
Margins not respected with tiny parts of glyphs located in margin

### ISSUE THREAD

**Issue by webservant317** [speedata/publisher#163]
When submitting my PDF files to Createspace I set my page margins to the minimum size of 0.25".  However, wherever Italicized text touched a right hand margin a small part of certain characters encroached on the margin.  This caused my Createspace submission to fail.

For a quick solution I may increase my margins a bit or remove the italic text.  Hopefully a better solution can be found in the Speedata software.

**Comment by webservant317** [comment]
Actually I see now that it may be any text.  So I don't know if this is possible to fix in Speedata, but I see that in order to avoid encroaching on the 0.25" margin I must set the margin a bit wider.

**Comment by pgundlach** [comment]
I'd need a sample page to be able to verify this. Could you provide one?

**Comment by pgundlach** [comment]
Perhaps this is related to italic text? 

```xml
<Layout xmlns:sd="urn:speedata:2009/publisher/functions/en"
  xmlns="urn:speedata.de:2009/publisher/en" >

  <Trace grid="yes"/>
  <Record element="data">
    <PlaceObject>
      <Textblock textformat="right">
        <Paragraph><I><Value>f</Value></I></Paragraph>
      </Textblock>
    </PlaceObject>
    
  </Record>
</Layout>
```

results in 

<img width="196" alt="bildschirmfoto 2018-03-04 um 19 24 42" src="https://user-images.githubusercontent.com/209434/36948935-0c90dec0-1fe2-11e8-86cd-c52977126e37.png">

The f's bounding box is smaller than its "ink width" and similar for italic g for the left margin.

I don't think there is much that I can do because it is a property of the font and I can't look inside the font.

**Comment by webservant317** [comment]
Patrick,
 
In the example I sited yes perhaps.
 
However, I have other examples related to other non-latin characters as well
that effect right, left, and top.  I didn't see the problem on the bottom.
 
Perhaps the problem cannot be fixed and one just needs to adjust the margin
accordingly.  I was able to solve my problem by expanding the margin, so
something to be aware of and perhaps document for future uses of Speedata.
 
Jeff

  _____
